impl/go.mod

```diff
@@ -71,6 +71,7 @@ require (
 	github.com/hyperledger/aries-framework-go/component/models v0.0.0-20230501135648-a9a7ad029347 // indirect
 	github.com/hyperledger/aries-framework-go/spi v0.0.0-20230427134832-0c9969493bd3 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/jellydator/ttlcache/v3 v3.1.0 // indirect
```

Comment:
Curious how you picked this among the [hundreds](https://awesome-go.com/caches/) of options?

Reply:
First google result! Is there one you suggest instead?

Reply:
Not really... I think I would have chosen https://pkg.go.dev/istio.io/pkg/cache just because I've heard of istio.